packages/@aws-cdk/aws-ecs/test/task-definition.test.ts

```diff
@@ -200,6 +200,36 @@ describe('task definition', () => {
       });
     });
 
+    test('A task definition for non windows instance with no memory config throws an error', () =>{
```

Comment:
this is a good test. 🎊

may I suggest adding another one with Windows OS here also? so we know the same values for windows are not throwing any errors 😬

Reply:
Gotcha added new test case with windows OS